src/ImageSharp/Formats/Gif/GifEncoderCore.cs

```diff
@@ -302,16 +312,10 @@ private void WriteGraphicalControlExtension<TPixel>(ImageFrame<TPixel> imageFram
         private void WriteGraphicalControlExtension<TPixel>(ImageBase<TPixel> image, IMetaData metaData, EndianBinaryWriter writer, int transparencyIndex)
             where TPixel : struct, IPixel<TPixel>
         {
-            // TODO: Check transparency logic.
```

Comment:
Original logic was incorrect. `DisposalMethod.Unspecified` is already the default and unless we are doing something crazy with the gif between decoding/encoding we should definitely copy the properties across. Devs have the power now to set each frame value independently should they choose.